src/pybind/mgr/cephadm/service_discovery.py

```diff
@@ -7,12 +7,13 @@ class Server:  # type: ignore
         pass
 
 import logging
-import tempfile
-from typing import Dict, List, TYPE_CHECKING, cast, Collection
 
-from orchestrator import OrchestratorError
+import orchestrator  # noqa
 from mgr_module import ServiceInfoT
-from mgr_util import verify_tls_files, build_url
+from mgr_util import build_url
+from typing import Dict, List, TYPE_CHECKING, cast, Collection
```

Comment:
It sounds as good improvement to avoid importing explicit types but I think it w'd break the style (in comparison with other cephadm files). So in case we go and introduce something like that I'd do it for all the files at once to keep the style similar across the code.